### PR TITLE
Homevork 2

### PR DESCRIPTION
**Pull request by Green-Mad**
Finish wy first git homework

### CODE REVIEW

File: HW_Task_2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#merger of companies
+#Eleks_employees
+#Toshiba_employees
+
+Eleks_employees = ['Freya', 'Jasmine', 'Mistress', 'Ariel', 'Kyrylo']
+Toshiba_employees = ['Zoreslav', 'Moses', 'Kyrylo', 'Lord', 'Marilouise']
+
+Toshiba_employees.extend(Eleks_employees)
+print(Toshiba_employees)
+
```
Comment (bohdandovbsyh) [inline HW_Task_2.py:10]: too many empty lines at the end of file)) you have 2 but only 1 required

also naming of variables. eleks_employees not Eleks

and eleks still have employees after your manipulations. please fix

File: HW_Task_3.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+casino_blacklist = {'Wolfgan Amadeus Mozart', 'Peter Griffin', 'Govard Lovekraft'}
+poker_blacklist = {'Wolfgan Amadeus Mozart', 'Matt Groening', 'Jack Bleck'}
+alcohol_blacklist ={'Wolfgan Amadeus Mozart', 'Spartak Subota', 'Govard Lovekraft'}
+half_bingo = casino_blacklist.intersection(poker_blacklist)
+bingo = half_bingo.intersection(alcohol_blacklist)
+print(bingo)
+
```
Comment (bohdandovbsyh) [inline HW_Task_3.py]: too many empty lines at the end of file)) you have 2 but only 1 required

File: HW_Task_4.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Omnivores = {'pigs', 'badgers', 'bears', 'foxes', 'coatis', 'civets'}
+Vegetarians = {'elephant', 'deer', 'giraffe', 'goat', 'sheep'}
```
Comment (bohdandovbsyh) [inline HW_Task_4.py]: naming. no capital letters in var names

File: HW_Task_5.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+VIP = {'Tony Stark', 'Bruse Wayne', 'Lex Luthor'}
+Common = ('Barry Alen', 'Peter Parker', 'Antman', None, None)
```
Comment (bohdandovbsyh) [inline HW_Task_5.py]: naming. no capital letters in var names

also why I can change VIP guests and can not add person to empty seat ?

File: HW_Task_6.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+non_unik_name = ['John', 'Marta', 'Alex', 'John', 'Sarah', 'Alex', 'Kate', 'Marta']
+unique_names_dict = {}
+for key in non_unik_name:
+    unique_names_dict[key] = None
+unique_names = list(unique_names_dict.keys())
+print(unique_names)
```
Comment (bohdandovbsyh) [inline HW_Task_6.py:4]: as I mentioned in the lesson you should use anly default functions of dict/set/list or tuple. not loops

File: HW_Task_2.py (hunk 1 above)
Comment (Green-Mad) [reply HW_Task_2.py:10]: Сорі, думав можна лишати більше одного)

File: HW_Task_6.py (hunk 5 above)
Comment (Green-Mad) [reply HW_Task_6.py:4]: Сорі, я цю задачу робив коли вже лупи були)

**Review by Green-Mad — COMMENTED** [review]
Fixed

**Review by Green-Mad — COMMENTED** [review]
Fixed

File: HW_Task_2.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+#merger of companies
+#Eleks_employees
+#Toshiba_employees
+
+eleks_employees = ['Freya', 'Jasmine', 'Mistress', 'Ariel', 'Kyrylo']
+toshiba_employees = ['Zoreslav', 'Moses', 'Kyrylo', 'Lord', 'Marilouise']
+
+toshiba_employees += eleks_employees
+eleks_employees = []
```
Comment (bohdandovbsyh) [inline HW_Task_2.py]: it is work. But not an optimal solution. we have special function for this

File: HW_Task_5.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+vip = {'Tony Stark', 'Bruse Wayne', 'Lex Luthor'}
```
Comment (bohdandovbsyh) [inline HW_Task_5.py]: I can change VIP guests

File: HW_Task_6.py (hunk 5 above)
Comment (bohdandovbsyh) [reply HW_Task_6.py:4]: Much better

File: HW_Task_2.py (hunk 6 above)
Comment (Green-Mad) [reply HW_Task_2.py]: Ти маєш на увазі так зробити?
toshiba_employees.extend(eleks_employees)
eleks_employees = []

File: HW_Task_5.py [hunk 8]
```diff
@@ -0,0 +1,4 @@
+vip = frozenset({'Tony Stark', 'Bruse Wayne', 'Lex Luthor'})
+common = ['Barry Alen', 'Peter Parker', 'Antman', None, None]
+print(vip)
+print(common)
```
Comment (bohdandovbsyh) [inline HW_Task_5.py:4]: no empty line at the end of file

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**